app/config/ConfigDecorator.scala

```diff
@@ -175,6 +175,9 @@ class ConfigDecorator @Inject() (
     if (lang.code equals "en") { "https://www.gov.uk/guidance/using-making-tax-digital-for-income-tax" }
     else { "https://www.gov.uk/guidance/using-making-tax-digital-for-income-tax.cy" }
 
+  def taxEnrolmentDeniedRedirect(url: Option[String]) =
+    s"https://www.tax.service.gov.uk/protect-tax-info?redirectUrl=$url"
```

Comment:
You cannot test this. Use app-config, See external-url section. So that in staging or else it uses the correct url.